src/utils/getNumberOfFileChanges.ts

```diff
@@ -0,0 +1,14 @@
+async function getNumberOfFileChanges(gitAPI:any, filePath:string){
+    let blames = await gitAPI?.repositories[0].blame(
+        filePath || "."
+    );
+    let commitHashArray = [] as String[];
+    let splitBlames = blames.split("\n").map((line:string) => {
+        commitHashArray.push(line.split(" ")[0]);
+    });
+
+    // get number of unique entries in commitHashArray
+    let uniqueCommitHashArray = [...new Set(commitHashArray)];
```

Comment:
Are we counting per commit or should we count per line change?
If this will be used on hover, I think line is better.

Reply:
It's not possible to do it per line change

Reply:
I'm not suggesting per line historical change, I suggest counting each change of each commit as one.
There's shock value and it's probably closer to the number of lines in a file, so the dev will be on edge.
https://github.com/watermelontools/wm-extension/pull/275#discussion_r887240266